hypha/apply/projects/templates/application_projects/includes/contracting_documents.html

```diff
@@ -50,22 +50,11 @@ <h5> {% trans "Are you sure you want to submit contracting documents?" %}</h5>
             </div>
             {% if contract_uploaded %}
             <div class="docs-block__row-inner">
-                <a class="docs-block__icon-link" href="{% url 'apply:projects:contract' pk=project.pk file_pk=contract.pk %}" target="_blank">
+                <a class="font-bold flex items-center w-auto text-left bg-white text-light-blue mr-0 mb-1 p-2.5 border-solid border border-light-blue" href="{% url 'apply:projects:contract' pk=project.pk file_pk=contract.pk %}" target="_blank">
```

Comment:
As the button is quite a re-usable UI element consider using `button button--white button--half-width` or it's variation to keep the look consistent. 

Also, the button styling misses focus and hover state. 

![Screenshot 2023-07-11 at 04 29 27@2x](https://github.com/HyphaApp/hypha/assets/236356/bc5b5290-5b84-4d19-a9e9-96ddbb49e0dd)

Reply:
I agree, we have project-actions--white as well but this is a custom button, we have a few other custom buttons as well like Upload, and Reupload contracts. These buttons look a bit different(more padding) from others just to get the user's attention.
 
Actually, these have to be links like other View options but these actions are slightly more important than links and less important than buttons.

I have added focus and hover state with bg and text colour change.